lib/main_common.pm

```diff
@@ -264,12 +264,15 @@ sub is_kernel_test {
 
 sub get_ltp_tag {
     my $tag = get_var('LTP_RUNTEST_TAG');
-    if (!defined $tag && defined get_var('HDD_1')) {
-        $tag = get_var('PUBLISH_HDD_1');
-        $tag = get_var('HDD_1') if (!defined $tag);
-        $tag = basename($tag);
-    } else {
-        $tag = get_var('DISTRI') . '-' . get_var('VERSION') . '-' . get_var('ARCH') . '-' . get_var('BUILD') . '-' . get_var('FLAVOR') . '@' . get_var('MACHINE');
+
+    if (!defined $tag) {
+        if (defined get_var('HDD_1')) {
+            $tag = get_var('PUBLISH_HDD_1');
+            $tag = get_var('HDD_1') if (!defined $tag);
```

Comment:
this `if` does not make sense , you can not get to this line when `$tag` is not defined ( see line 269 )

Reply:
Do you mean 268? The meaning is define `$tag` when not defined via `LTP_RUNTEST_TAG`. So `my $tag = get_var('LTP_RUNTEST_TAG'); if (!defined $tag) { ... }` construct is IMHO correct. Or am I missing something?

Reply:
The question is, whether keep `LTP_RUNTEST_TAG`. IMHO we have never used it.

Reply:
no, you are right , that's me who was missing :) sorry